source/common/http/headers.h

```diff
@@ -69,6 +69,9 @@ class CustomHeaderValues {
   const LowerCaseString Pragma{"pragma"};
   const LowerCaseString Referer{"referer"};
   const LowerCaseString Vary{"vary"};
+  const LowerCaseString XRateLimitLimit{"x-ratelimit-limit"};
+  const LowerCaseString XRateLimitRemaining{"x-ratelimit-remaining"};
+  const LowerCaseString XRateLimitReset{"x-ratelimit-reset"};
```

Comment:
Can you move the changes in the file into filter specific code? I don't think it needs to be here.